packages/vuetify/src/util/__tests__/dateTimeUtils.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { weekNumber } from '../dateTimeUtils'
+
+describe('/util/dateTimeUtils.ts', () => {
+  it('should return correct weeknumber', () => {
+    expect(weekNumber(2010, 0, 1, 1, 4)).toBe(53)
```

Comment:
Could this be a collection of iterated arrays to reduce the spec size?

```ts
it('should return correct weeknumber', () => {
  for (const [val, ...args] of [
    [53, 2010, 0, 1, 1, 4],
    [53, 2016, 0, 1, 0, 4],
    ...etc
  ]) {
    expect(weekNumber(...args).toBe(val)
  }
})
```

Reply:
https://jestjs.io/docs/en/api#testeachtablename-fn-timeout

Reply:
Used suggested test.each